include/wgvk_structs_impl.h

```diff
@@ -1493,7 +1493,9 @@ typedef struct RenderPassCommandBeginOcclusionQuery {
 } RenderPassCommandBeginOcclusionQuery;
 
 typedef struct RenderPassCommandEndOcclusionQuery {
-    // This command takes no arguments.
+    // Empty structs are technically illegal so a placeholder is needed
```

Comment:
They are actually a GNU extension in C, documented here:
https://gcc.gnu.org/onlinedocs/gcc/Empty-Structures
I would say "nonstandard" instead of "illegal".

Reply:
Actually, saying they have zero size in C++ makes me curious if this might cause ABI problems. I expect dawn or whatever is already doing this with the member, so probably fixes some bug too.